executor/executor.go

```diff
@@ -440,7 +440,7 @@ func (e *SelectLockExec) Next(goCtx goctx.Context) (Row, error) {
 				return nil, errors.Trace(err)
 			}
 			// This operation is only for schema validator check.
-			txnCtx.UpdateDeltaForTable(id, 0, 0)
+			txnCtx.GetTableDelta(id)
```

Comment:
I see only `updateTableDelta` and here use this function. We cannot know that it will insert table `id` if we don't find it.
I think we should use two functions called `get` and `insert(id, *TableDeta)`, which may be cleaner.